aten/src/ATen/core/TensorImpl.h

```diff
@@ -798,7 +787,7 @@ struct CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
   void set_sizes_contiguous(at::IntList new_size) {
     AT_ASSERT(!is_variable());
     auto old_dim = sizes_.size();
-    sizes_ = new_size.vec();
+    sizes_ = new_size;
```

Comment:
@smessmer police!! Given how much work you did making `IntList` not implicitly convertible to vector, you probably don't want it implicitly convertible to small vector, do you?

Reply:
Does this implicitly convert new_size to a SmallVector and then do assignment?